docs/components/SegmentedControl/Web.stories.tsx

```diff
@@ -46,34 +41,14 @@ export const Controlled: Story = () => {
   );
 };
 
-export const AsUncontrolled: Story = () => {
-  const [activeOption, setActiveOption] = useState("calendar");
-
-  const handleChange: React.ChangeEventHandler<HTMLInputElement> = e => {
-    setActiveOption(e.target.value);
-  };
-
+export const BasicUncontrolled: Story = () => {
   return (
-    <SegmentedControlContext.Provider
-      value={{ handleChange, selectedOption: activeOption }}
-    >
-      <SegmentedControl.Base>
-        <SegmentedControl.Option value="calendar" ariaLabel="Calendar">
-          <Icon name="calendar" />
-        </SegmentedControl.Option>
-        <SegmentedControl.Option value="phone" ariaLabel="Phone">
-          <Icon name="phone" />
-        </SegmentedControl.Option>
-        <SegmentedControl.Option value="presentation" ariaLabel="Presentation">
-          <Icon name="presentation" />
-        </SegmentedControl.Option>
-        <SegmentedControl.Option value="availability" ariaLabel="Availability">
-          <Icon name="availability" />
-        </SegmentedControl.Option>
-        <SegmentedControl.Option value="chat" ariaLabel="Chat">
-          <Icon name="chat" />
+    <SegmentedControl defaultOption="pizza">
```

Comment:
Nit: If the option has a prop called `value` favour sharing the language to make it clear the connection between the two. `defaultValue`